pbj-integration-tests/src/test/java/com/hedera/pbj/intergration/test/Issue96Test.java

```diff
@@ -0,0 +1,281 @@
+package com.hedera.pbj.intergration.test;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.hedera.pbj.runtime.ProtoParserTools;
+import com.hedera.pbj.runtime.io.buffer.BufferedData;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import com.hedera.pbj.test.proto.pbj.Everything;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.HexFormat;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+final class Issue96Test {
```

Comment:
Would be nice to give this a name that includes what issue 96 is about like TruncatedDataTests as issue 96 will not mean anything when coming back and looking at the code in the future.

Reply:
Yes... I'm really bad with names. It just started as a bug test and grown into more. Changed the name as suggested.